share_extension/common/extension_channel_item.js

```diff
@@ -75,7 +75,7 @@ export default class ExtensionChannelItem extends PureComponent {
                             ellipsizeMode='tail'
                             numberOfLines={1}
                         >
-                            {channel.display_name}
+                            {channel.display_name || channel.name}
```

Comment:
Don't channels always have a display name?

Reply:
oh this is some random test that I added, I'll fix this